repos/system_upgrade/el7toel8/actors/checkipaserver/libraries/library.py

```diff
@@ -0,0 +1,61 @@
+from leapp import reporting
+
+
+REPORT_COMMON = [
+    reporting.Tags([reporting.Tags.SERVICES]),
+    reporting.ExternalLink(
+        url=(
+            "https://access.redhat.com/documentation/en-us"
+            "/red_hat_enterprise_linux/8/html/installing_identity_management"
+            "/migrate-7-to-8_migrating"
+        ),
+        title="Migrating IdM from RHEL 7 to 8",
+    ),
+    reporting.RelatedResource("package", "ipa-server"),
+]
+
+
+def ipa_inhibit_upgrade(ipainfo):
+    """Create upgrade inhibitor for configured ipa-server
+    """
+    entries = [
+        reporting.Title(
+            "ipa-server does not support in-place upgrade from RHEL 7 to 8"
+        ),
+        reporting.Summary(
+            "An IdM server installation was detected on the system. IdM "
+            "does not support in-place upgrade to RHEL 8. Please follow "
+            "the migration guide lines."
+        ),
+        reporting.Remediation(
+            hint="Please follow the IdM RHEL 7 to 8 migration guide lines."
+        ),
+        reporting.Severity(reporting.Severity.HIGH),
+        reporting.Flags([reporting.Flags.INHIBITOR]),
+    ]
+    entries.extend(REPORT_COMMON)
+    return reporting.create_report(entries)
+
+
+def ipa_warn_pkg_installed(ipainfo):
+    """Warn that unused ipa-server package is installed
+    """
+    if ipainfo.is_client_configured:
+        summary = (
+            "The ipa-server package is installed but only IdM client is "
+            "configured on this system."
+        )
+    else:
+        summary = (
+            "The ipa-server package is installed but neither IdM server "
+            "nor client is configured on this system."
+        )
+    entries = [
+        reporting.Title("ipa-server package is installed"),
```

Comment:
title is not telling user anything about why you create the report. be little more specific